### ISSUE TITLE
Fix: Update pto channel message approve deny response

### ISSUE THREAD

**Issue by PatrickDohn** [f1v/hrbot#219]
**Summary**
_A clear and concise description of what the chore is._

**Acceptance Criteria**

1. Once an event is approved or denied the message should update and say that the event was approved or denied.
2.
3.

**Time Estimation (in hours)**

```

```

**Checklist**

- [ ] Chore must have associated Milestone
- [ ] Chore must have associated Project if PR is merged
- [ ] Chore must be one of three types below

#### Chore Types and Definitions

```
Chore - A catch all for uncategorized chores
TechDebt - A technical chore that provides no functional change but cleans up code
Task - A non-technical chore that involves work outside of code
```

**Comment by davidholyko** [comment]
duplicate of https://github.com/f1v/hrbot/issues/218
